package.json

```diff
@@ -250,6 +250,15 @@
           "invoice.item.pay"
         ]
       },
+      {
+        "permissionName": "ui-invoice.cancel",
+        "displayName": "Invoice: Cancel invoices",
```

Comment:
permissions should be translated as well

Reply:
done